src/components/data/toolbar/TrashMenu.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @author sriram
+ *
+ * A button with menu items for DS trash operations.
+ *
+ */
+
+import React, { useState } from "react";
+import { useTranslation } from "i18n";
+
+import ids from "../ids";
+import styles from "../styles";
+
+import { build } from "@cyverse-de/ui-lib";
+import {
+    Button,
+    makeStyles,
+    Menu,
+    ListItemIcon,
+    ListItemText,
+    MenuItem,
+} from "@material-ui/core";
+
+import {
+    ArrowDropDown,
+    Delete,
+    DeleteForever,
+    RestoreFromTrash,
+    DeleteSweep,
+} from "@material-ui/icons";
+import TrashMenuItems from "./TrashMenuItems";
+
+const useStyles = makeStyles(styles);
+
+export function TrashMenu(props) {
+    const {
+        baseId,
+        selected,
+        handleDelete,
+        handleRestore,
+        handleEmptyTrash,
+    } = props;
+    const { t } = useTranslation("data");
+    const classes = useStyles();
+
+    const [trashAnchor, setTrashAnchor] = useState(null);
+
+    const onTrashClose = () => {
+        setTrashAnchor(null);
+    };
+
+    const onTrashMenuClick = (event) => {
+        setTrashAnchor(event.currentTarget);
+    };
+
+    return (
+        <>
+            <Button
+                id={build(baseId, "trashButton")}
+                variant="outlined"
+                size="small"
+                disableElevation
+                color="primary"
+                className={classes.button}
+                onClick={onTrashMenuClick}
+                aria-haspopup={true}
+                aria-controls={baseId}
```

Comment:
I think `baseId` in this context is the toolbar ID?  I think `aria-controls` is meant to have the ID of the element this opens/controls.  In this case, this should be the ID of the `Menu` below.